package.json

```diff
@@ -15,6 +15,7 @@
     "url": "git@github.com:nightwatchjs/nightwatch.git"
   },
   "dependencies": {
+    "@cucumber/cucumber": "^7.3.1",
```

Comment:
We need to make this an optional dependency. the overall users of nightwatch who use cucumber I think it will be fairly small.